doc/build-helpers/testers.chapter.md

```diff
@@ -165,6 +165,67 @@ testers.shellcheck {
 A derivation that runs `shellcheck` on the given script(s).
 The build will fail if `shellcheck` finds any issues.
 
+## `shfmt` {#tester-shfmt}
+
+Runs files through `shfmt`, a shell script formatter.
```

Comment:
So is this a test? Or a file transforming function? :smiley:

Reply:
Good catch! Addressed in the latest.